wlct/cogs/clot.py

```diff
@@ -26,7 +26,6 @@ def __init__(self, bot):
                     print("Saving discord id: {} for user".format(ctx.message.author.id))
                     discord_obj = DiscordUser(memberid=ctx.message.author.id)
                     discord_obj.save()
-                    player.discord = discord_obj
```

Comment:
This needs to be player.discord_member now in order to link properly.

Reply:
Done